demo/netops-demo/templates/_helpers.tpl

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- define "generateName" -}}
+{{- "generate" | trunc 63 -}}
```

Comment:
`trunc` what? it's a fix name. It should be in Helm convention format (`.Chart.Name` or `.Release.Name` prefix)